js/src/views/Account/Header/header.js

```diff
@@ -67,6 +71,10 @@ export default class Header extends Component {
             <Balance
               account={ account }
               balance={ balance } />
+            <Certifications
+              certifications={ certifications }
+              dappsUrl={ api.dappsUrl }
```

Comment:
The idea is to have smart & dumb components or at least move in this direction, as it is considered best practice. If `Certifications` is really only going to be used in this place, I will move it. Otherwise (e.g. when it's used in `Accounts`) it makes sense to keep `Certifications` dumb.